greyhounds/zoya.md

```diff
@@ -5,4 +5,14 @@ greyhound: zoya
 permalink: /greyhounds/zoya/
 ---
 
-I've just arrived!
+Hi! My name is Zoya! 
+
+I've recently retired from the fast-paced world of being a professional greyhound, and I love lounging, snuggling, and leaning against the nearest human. I'm not especially food-motivated, but I sure wasn't kidding about snuggling-- nothing makes me feel safer and happier than curling up next to you for some scritches and pets, or lying down for a nice long brushing.
```

Comment:
Looks like our spell checker doesn't believe scritches is a word. Clearly, the spell checker is wrong :wink: 

I'll add this to our list of acceptable words.